File: app/src/main/java/volovyk/guerrillamail/data/remote/pojo/GetEmailAddressResponse.java
package volovyk.guerrillamail.data.remote.pojo;

import com.google.gson.annotations.SerializedName;

public class GetEmailAddressResponse {
    @SerializedName("email_addr")
    private String emailAddress;
    @SerializedName("sid_token")
    private String sidToken;

    public String getEmailAddress() {
        return emailAddress;
    }

    public String getSidToken() {
        return sidToken;
    }
}

File: app/src/main/java/volovyk/guerrillamail/data/remote/pojo/GetEmailAddressResponse.kt
package volovyk.guerrillamail.data.remote.pojo

import com.google.gson.annotations.SerializedName

class GetEmailAddressResponse {
    @SerializedName("email_addr")
    val emailAddress: String? = null

    @SerializedName("sid_token")
    val sidToken: String? = null
}